matrix/common/aws/batch_handler.py

```diff
@@ -18,8 +18,8 @@ def __init__(self):
         self.job_def_arn = os.environ['BATCH_CONVERTER_JOB_DEFINITION_ARN']
 
         self._client = boto3.client("batch", region_name=os.environ['AWS_DEFAULT_REGION'])
-        self._cloudwatch_handler = CloudwatchHandler()
 
+    @retry(reraise=True, wait=wait_fixed(2), stop=stop_after_attempt(5))
```

Comment:
What happened before if the enqueuing failed?

Reply:
exception raised that bubbled up to query runner